File: dollarx/src/main/java/info/dollarx/custommatchers/hamcrest/CustomMatchersUtil.java
package info.dollarx.custommatchers.hamcrest;

import info.dollarx.Path;

final class CustomMatchersUtil {


    static public String wrap(Path el) {
        String asString = el.toString();
        return (asString.contains(" ")) ? String.format("(%s)", asString) : asString;
    }
}

File: dollarx/src/main/java/com/github/loyada/dollarx/custommatchers/hamcrest/CustomMatchersUtil.scala
package com.github.loyada.dollarx.custommatchers.hamcrest

import com.github.loyada.dollarx.Path

object CustomMatchersUtil {
  def wrap(el: Path): String = {
    val asString: String = el.toString
    return if ((asString.contains(" "))) String.format("(%s)", asString) else asString
  }
}
